apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater/services/fetchOrderPrice.ts

```diff
@@ -0,0 +1,33 @@
+import { AccountAddress, QuoteResults, SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { tradingSdk } from 'tradingSdk/tradingSdk'
+
+import { getRemainderAmount } from 'legacy/state/orders/utils'
+
+import { OrderToCheckFillability } from '../types'
+
+export async function fetchOrderPrice(
+  chainId: SupportedChainId,
+  order: OrderToCheckFillability,
+): Promise<QuoteResults | null> {
+  const amount = getRemainderAmount(order.kind, order)
+
+  try {
+    return tradingSdk
```

Comment:
Since this is a promise, shouldn't `await` here so the catch can work?
Unless you want the promise to bubble up, and only capture if anything here breaks up.